Feature/profile redesign

aditilamkhede opened the pull request:
### What does this pull request aim to achieve? 💯
1. Navigation Menu for desktop and mobile


_Please be concise and link any related issue(s):_ 


<!--### 🚨Before submitting this pull request🚨:-->

<!--_Please do **NOT** submit this PR if you have not done the following:_-->

- [ ] I have checked that no one else is working on similar changes.
- [ ] I have read the latest [**README**](README.md) and understand the development workflow.
- [ ] The name of this branch is: **`feature/<branch_name>`** for a regular feature, **`hotfix/<branch_name>`** for a hotfix, and **`service/<branch_name>`** for a microservice, where `<branch_name>` briefly describes the issue(s) resolved and is prefixed with the issue number(s) (e.g. `1127-1130-update-git-branching-model`). The branch is created in a cloned version of this repo, **not a forked repo**.
- [ ] This branch is rebased/merged with the **latest staging** (or the **latest production** if it is a hotfix).
- [ ] There are no merge conflicts.
- [ ] There are no console warnings and errors.
- [ ] On the right hand side of this PR, I have linked the appropriate issue(s) under "Issues" and added a project progress under "Projects".
- [ ] The title describes the issue(s) being addressed using format: **`<issue_numbers> - <brief_issue_description>`** (e.g. `#1127/#1130 - Update Git Branching Model`)

clubkemp committed:
creating a basic tabs component expecting props to control the display. Creating Profiletabs to style basic tabs for use in profile section. Adding ProfileTabs as main navbar option for mobile view. State view will be used to control the components rendered to the page

clubkemp committed:
creating a SideMenu component that uses the menu wrapper from the feed to create a similar looking menu. This component will function in a similar fashion to the ProfileTabs component where it takes an array of values and a few other props to create the menu

clubkemp committed:
updating en_US to include new menu tab items. Adding tabs array using those translations

clubkemp committed:
fixing a bug for the scrolling of the tabs when under desktop size

linadinh2602 committed:
Add PostTabCard

aditilamkhede committed:
See All Component

See All Component  and see all page for mobile view.

linadinh2602 committed:
revert change on post

linadinh2602 committed:
add end of line

clubkemp committed:
removing console log and unused useEffect

clubkemp committed:
Merge branch 'staging' of https://github.com/FightPandemics/FightPandemics into feature/profileTabs

clubkemp committed:
refactor BasicTabs to take in an array that passes through the position, default view, the tabs data as an array that contains the view itself as a nested react component

aditilamkhede committed:
Merge branch 'staging' into feature/profile-redesign-see-all

aditilamkhede committed:
Update SeeAllComponent.js

clubkemp committed:
updating basicTabs to use object

clubkemp committed:
removing if bloc

clubkemp committed:
Merge pull request #2217 from FightPandemics/feature/profileTabs

Feature/profile tabs

aditilamkhede committed:
Merge branch 'feature/profile-redesign' into feature/profile-redesign-see-all

clubkemp committed:
merging in main profile-redesign branch

clubkemp committed:
adding utility function to check for screen size. Adding ternary statement to tabView object to change position based on screen size. Tweaking profileTabs styles to change slightly based on top position or not

clubkemp committed:
arbritrary space for upload uncommit

clubkemp committed:
still trying for the uploadPic uncommit

clubkemp committed:
still trying for the uploadPic uncommit

clubkemp committed:
disabling future tabs

linadinh2602 committed:
response to review

clubkemp committed:
changing the diabled attr of our views object to disabled: true/false to make it more intuative. Creating a isSelfViews function that checks if a user is logged in and modifies the tabViews object accordingling

clubkemp committed:
updating the position logic of tabs to use the mq from theme constant

linadinh2602 committed:
Merge pull request #2219 from FightPandemics/feature/ProfileCardComponent

#2200 - Add Redesign PostTabCard Component

linadinh2602 committed:
add Dropdown functionality for contact meny

aditilamkhede committed:
See All Component with Desktop view Integration

aditilamkhede committed:
Merge branch 'feature/desktop-view-tabs' into feature/profile-redesign

aditilamkhede committed:
Merge branch 'feature/profile-redesign' into feature/profile-redesign-see-all

linadinh2602 committed:
Merge pull request #2226 from FightPandemics/feature/add-dropdown

#2200 - Add Redesign PostTabCard Component's DropDown Functionality

clubkemp committed:
changing the logic for altering the menu items to be performed in a useEffect and menu held in state

clubkemp committed:
menu building and displaying. time to style

clubkemp committed:
adding some styling for the mobile component

aditilamkhede committed:
Merge branch 'staging' into feature/profile-redesign-see-all

clubkemp committed:
desktop styling in place

clubkemp committed:
moving the create menu out of a useEffect

clubkemp committed:
adding translations

clubkemp committed:
loadMOre

clubkemp committed:
loadMenu

clubkemp committed:
adding an isSelf dependency to building the navs and moving all that logic to a helper funcion. linter likes it

clubkemp committed:
fixing cap issue on the tras json

clubkemp committed:
ternary statements for mobile/desktop views

clubkemp committed:
adding the exhaustive-deps ignore

clubkemp committed:
disable eslint on buildNavMenu

aleksandra-b-t committed:
user-profile-card-redesing

clubkemp committed:
addressing review concerns

aditilamkhede committed:
Merge branch 'feature/profile-redesign' into feature/profile-redesign-see-all

aditilamkhede commented:
@SinghRomika test here. https://featureprofile-redesign.fightpandemics.xyz/

aditilamkhede committed:
Merge branch 'feature/profile-redesign' into feature/profile-redesign-see-all

SinghRomika commented:
@aditilamkhede Can you plz share the requirements for this PR#2243. Thanks!

SinghRomika reviewed (dismissed):
@aditilamkhede @clubkemp Reviewed the Desktop and Mobile profiles redesign the navigation menu  works as expected
Request Menu tab and offer menu tab are active and clickable.
Active, Organization, Thanks, Badges menu tabs are currently inactive and not clickable.
**Desktop Review**
Browser: Microsoft Edge and Chrome
https://user-images.githubusercontent.com/78573508/109247252-7533de80-7798-11eb-8ea5-108d8a917c2d.mp4

**Mobile Review**
Mobile: Android
Browser: Chrome
https://user-images.githubusercontent.com/78573508/109247291-8381fa80-7798-11eb-8b2d-3705a67f51cf.mp4

aditilamkhede committed:
Integration with new menu for desktop view

Integration with new menu for desktop view
-- Intergrated See All component for Requests/Offers/Posts parents tabs on profile page

aditilamkhede committed:
added wrapper to wrap menu and component

aditilamkhede committed:
set the visibility of component and desktop menu as per device.

aditilamkhede committed:
Update Profile.js

clubkemp committed:
linting issue...

clubkemp committed:
changing widths

clubkemp committed:
fixing merge conflicts

clubkemp committed:
removing my activity section header

aditilamkhede committed:
cleanup code

aditilamkhede committed:
Merge branch 'staging' into feature/profile-redesign-see-all

clubkemp committed:
mergin conflicts

clubkemp committed:
adding ternary logic to disable draft tabs

clubkemp committed:
removing commented out section

aditilamkhede committed:
disabled drafts tab

aditilamkhede committed:
Merge branch 'feature/profile-redesign-see-all' into feature/profile-redesign-menu-integration

aditilamkhede committed:
Merge pull request #2244 from FightPandemics/feature/profile-redesign-menu-integration

Feature/profile redesign menu integration

aditilamkhede committed:
Update SeeAllPost.js

aleksandra-b-t committed:
required changes

aleksandra-b-t committed:
final profile card redesign changes

I added last changes of message button (for mobile and desktop). For now the thank button has been removed.

aditilamkhede committed:
cleanup

aditilamkhede committed:
Merge pull request #2221 from FightPandemics/feature/profile-redesign-see-all

See All Component for Mobile view and Desktop View

ddveloper commented on backend/lib/endpoints/posts.js:
```diff
@@ -52,18 +52,30 @@ async function routes(app) {
       ];
+      // "IA" stands for InActive or Archived
+      //In future we will also have "D" for Drafts
+      if(postMode === 'IA'){
```
this would be a good use case of enum type, and it's better to let the code explain itself (e.g. define an enum of POST_MODE = ['inactive', 'draft'] inside the `lib/endpoints/schema/posts.js`, so the input can also be validated)

ddveloper commented on backend/lib/endpoints/schema/posts.js:
```diff
@@ -18,6 +18,7 @@ const getPostsSchema = {
     .prop("ignoreUserLocation", S.boolean().default(false))
     .prop("objective", S.string().enum(POST_OBJECTIVES))
     .prop("skip", S.integer())
+    .prop("postMode", S.string())
```
here can use the enum POST_MODE defined above to validate input

ddveloper replied:
like mentioned above, using a enum type can help to validate the input here in the schema

ddveloper commented on client/src/components/Feed/StyledPostTabCard.js:
```diff
@@ -0,0 +1,102 @@
+export const StyledUpdateAt = styled.p`
+  font-size: ${xxsmall};
+  line-height: 1.4rem;
+  color: #939393;
```
hard-coded color could be replaced with the constant (e.g. DARK_GRAY = "#939393")

ddveloper reviewed (commented):
Great team work 👍  thank you all ❤️ 
I added one comment about backend enum type and a comment about color constant

aleksandra-b-t committed:
user profile view changes

When no one is logged.

aleksandra-b-t committed:
small changes

user info card in auto height.

clubkemp committed:
adding logic to turn see all components tab types into translation variable

clubkemp committed:
adding views translations to en_GB json

rabbi08 reviewed (changes requested):
Thanks @aditilamkhede @linadinh2602 @clubkemp 

I have following observations:
1. The remote feature is not displayed on the post, even though the feature exist in Filter. Please see below screenshot.
<img width="1440" alt="Screen Shot 2021-03-04 at 12 45 29 AM" src="https://user-images.githubusercontent.com/67168400/109917577-9490a700-7c83-11eb-9687-155b1da8a5d9.png">

2. The 'Activity' tab is BLANK for the user profile, this looks like user have not done anything. We can display all the posts or the recent activity under 'Activity' tab but it should not be blank.

3. Unable to create draft, BUT, I guess, the 'Draft' tab is not the part of this feature. 
Hence, great if you can confirm that this feature is not being developed/planned yet.
AND if we are not planning to have the 'Draft' feature ready, can it be to hidden in the UI? so that users should not get confuse, like how this draft works,, or so?

Please see below GIF for Issue#2 & 3:
![ezgif com-video-to-gif-2](https://user-images.githubusercontent.com/67168400/109918717-7166f700-7c85-11eb-9bfa-5a062ca27de1.gif)

4. On Mobile: Please note its my same account I used for testing in Desktop and mobile. I can see my post on desktop but not on mobile:

My existing post are not displayed under any "Activity/Offer/Request" tab. Please see below GIF:
![ezgif com-video-to-gif-3](https://user-images.githubusercontent.com/67168400/109918697-6a3fe900-7c85-11eb-923a-76f2060f2ba5.gif)

aleksandra-b-t committed:
change shadow

aditilamkhede committed:
DekstopView component for Posts

Created new desktop view component for desktop view of profile.

aditilamkhede committed:
Delete posts added

linadinh2602 committed:
Merge pull request #2258 from FightPandemics/feature/profile-redesign-see-all

Feature/profile redesign see all

clubkemp committed:
adding gtm tags to menu tabs

clubkemp committed:
Merge branch 'feature/profile-redesign' of https://github.com/FightPandemics/FightPandemics into feature/profile-redesign

clubkemp committed:
adding gtm map to the profileDesktop component

aditilamkhede commented on client/src/constants/gtm-tags.js:
```diff
@@ -86,8 +86,18 @@ module.exports = {
     checkBadge: "_AV_CB",
-    requests: "_RQS",
-    offers: "_OFS",
+    requests: "_RQ",
```
change this to _RE

sarahjliu replied:
Will removing RQS, OFS from this file affect existing GTM tags that might be using them?

linadinh2602 committed:
Integrate mobile post tab card, refactor code to add cached the posts

clubkemp committed:
Merge branch 'staging' of https://github.com/FightPandemics/FightPandemics into feature/profile-redesign-bugfixes

clubkemp committed:
adding the remote and in person span back in post

clubkemp committed:
removing temp file for comparing

clubkemp committed:
fixing typo in translation file for wasDeleted

clubkemp committed:
fixing gtm tag mixup, request to _RE instead of _RQ and making sure requests-archieved is pulling request GTM instead of OFFERS

clubkemp committed:
adding logic to break the menu build if the isSelf is undefined

clubkemp committed:
commenting out drafts until ready for future dev

aditilamkhede committed:
Merge branch 'feature/profile-redesign' into feature/profile-redesign-user-profile-card

linadinh2602 committed:
fix loading issue

aditilamkhede committed:
Merge pull request #2264 from FightPandemics/feature/profile-redesign-bugfixes

Feature/profile redesign bugfixes

aleksandra-b-t committed:
mobile location fixed

linadinh2602 committed:
Allow create post

linadinh2602 committed:
fix other issues

aditilamkhede committed:
Added gtmid for desktop view posts

linadinh2602 committed:
make it cleaner

linadinh2602 committed:
deleted console

linadinh2602 committed:
fix small bug issue with userId and login error

aleksandra-b-t committed:
changes px to rem

linadinh2602 committed:
fix race condition in initializing menu

linadinh2602 committed:
remove console log

linadinh2602 committed:
Merge pull request #2263 from FightPandemics/feature/integrate-posttabcard-mobile-view

#2262 - Integrate PostTabCard mobile view in Profile Page

clubkemp committed:
translations in place for the reqests, offers, and posts tabs

aleksandra-b-t committed:
Merge branch 'feature/profile-redesign' into feature/profile-redesign-user-profile-card

clubkemp committed:
Merge pull request #2241 from FightPandemics/feature/profile-redesign-user-profile-card

Looks Good!

clubkemp committed:
adding DET affix to the desktop internal tabs. Adding a TagManager event to fire on internal tab clicks for mobile view cards

rabbi08 reviewed (changes requested):
Thanks @aditilamkhede @clubkemp @linadinh2602 

I have following observations:

1. 'Message' icon:
On Desktop:
- On the HelpBoard-> The 'message' icon is NOT displayed for any post . Please see below GIF.
![ezgif com-video-to-gif](https://user-images.githubusercontent.com/67168400/111254540-b24afe00-85eb-11eb-9b4a-616b524060de.gif)

But on mobile -> the 'message' icon is not displayed properly. Please see the screenshot below as compared to production environment.

<img width="750" alt="Screen Shot 2021-03-16 at 12 01 33 AM" src="https://user-images.githubusercontent.com/67168400/111254246-fee20980-85ea-11eb-9c71-80bb1311fe3a.png">

2. 'See All' feature:
- Whle testing on mobile:
'See All' option is displayed even when there is no post to display. 'See All' option should be displayed and enabled/clickable if there are more post to be displayed. 

- 'See All' button is NOT clickable , as for 'Archive' I do have post to 'See All', it snot working.
Please see below GIF for above issues.
![ezgif com-video-to-gif-2](https://user-images.githubusercontent.com/67168400/111254713-01912e80-85ec-11eb-942b-2cd709f9bec9.gif)

aditilamkhede committed:
Linking see-all to mobile view

linadinh2602 committed:
Add current tab

rabbi08 commented:
Adding another observations:
- When user navigate to Post detail page from 'Archive' TAB and then click 'BACK' button on post detail page
Issue: User lands on 'Active' TAB of the user profile BUT not 'Archive' TAB from where user navigate to Post Detail page.

This is 'BACK' button navigation issue that it should land on the same page/TAB from where they navigate. This issue is not high priority to fix, but great if we can do that as keeping the bugs open at this stage of FP might result in that those bugs never get fixed.

Thanks.

aditilamkhede committed:
Merge pull request #2276 from FightPandemics/feature/fix_some_bugs

#2262 - Add current tab, remove see all when there is no posts

ddveloper commented on backend/lib/endpoints/posts.js:
```diff
@@ -52,18 +52,33 @@ async function routes(app) {
       ];
+      // "IA" stands for InActive or Archived
+      //In future we will also have "D" for Drafts
+      if (postMode === "IA") {
```
I think it's better to use enum for this input, e.g. POST_MODE, [this one](https://github.com/FightPandemics/FightPandemics/blob/7190df6f87b09bcd7fa26e4d1d348ca095605c46/backend/lib/models/Post.js#L4) is a reference

ddveloper replied:
currently this filter is different from above one only for the Date part, maybe use a conditional statement to simplify it, like `postMode === 'IA' ? $lt ... : $gt ...

aditilamkhede replied:
> currently this filter is different from above one only for the Date part, maybe use a conditional statement to simplify it, like `postMode === 'IA' ? $lt ... : $gt ...

This is Done too.

ddveloper commented on backend/lib/endpoints/posts.js:
```diff
@@ -113,15 +128,30 @@ async function routes(app) {
+            ],
+          });
+        } else if (workModes.includes("remote")) {
+          filters.push({
```
this `else if` filter is different from above one only at the $nin and $in part, maybe use a conditional statement to simplify it

aditilamkhede replied:
Done.

ddveloper reviewed (commented):
I add some comment on the backend 🍪

clubkemp committed:
adding a if logic function to check if coming from profile, this builds the html differently and applys a different message container

aditilamkhede committed:
See-all changes

clubkemp committed:
merging to main feature branch

aditilamkhede committed:
commented postCount

hiran11 commented:
@aditilamkhede @clubkemp @linadinh2602 

1) In the Profile page i could see "No post to display" alignment has to be fixed.

 
![image](https://user-images.githubusercontent.com/51660881/111376653-1b775380-8676-11eb-847e-ef0c70824e8d.png)


2) I could see blank space after the last post in the Request option  before reaching the end of page. Please look at the link for reference.


https://user-images.githubusercontent.com/51660881/111379989-29c76e80-867a-11eb-8b41-38dab4ef5960.mp4



![Screenshot (126)](https://user-images.githubusercontent.com/51660881/111379489-942bdf00-8679-11eb-9a28-912309e3bf80.png)


Mobile version 

Alignment of "No Posts matching your criteria " has to be fixed.

![Screenshot_20210316-173419 (1)](https://user-images.githubusercontent.com/51660881/111384418-d9eba600-867f-11eb-8f86-cb2a4c044a4d.png)

linadinh2602 committed:
prevent fetch from calling duplicate

linadinh2602 committed:
remove console

linadinh2602 committed:
fix post count

linadinh2602 committed:
remove comment

clubkemp committed:
applying some ternary logic to dry out code

AndersonNY commented:
<img width="388" alt="Screen Shot 2021-03-16 at 4 53 17 PM" src="https://user-images.githubusercontent.com/56901619/111379500-97bf6600-8679-11eb-82ea-443cb0f0ead6.png"> This is mobile version checking with iPhone 11

As you see the other buttons are not clickable.

hiran11 commented:
@aditilamkhede 
mobile version 

When I try to click the "see all post" under the offers option I could see the request goes back to request option with "No matching post" displayed .

https://user-images.githubusercontent.com/51660881/111388419-9ac05380-8685-11eb-81f9-87beab7f56e5.mp4

aditilamkhede committed:
Merge pull request #2279 from FightPandemics/feature/profile-redesign-user-profile-card

Feature/profile redesign user profile card

linadinh2602 committed:
fix internal tab not set

aditilamkhede committed:
Merge pull request #2281 from FightPandemics/feature/profile-redesign-backendFix

applying some ternary logic to dry out code

aditilamkhede commented:
> @aditilamkhede
> mobile version
> 
> When I try to click the "see all post" under the offers option I could see the request goes back to request option with "No matching post" displayed .
> 
>  03_16_2021_05_57_PM.mp4

That is the correct behavior. What See-all does is, it shows all the posts for Top Navigation Menu selection. In your example you have selected Post Tab on Top then in your internal Tabs are Request and Offer. When you click on see-all it should show detailed view of Post Menu. Because Request tab doesn't have any posts, when you go to see-all page it shows message saying No posts matching your criteria. We can may be change the message.

aditilamkhede commented:
> <img alt="Screen Shot 2021-03-16 at 4 53 17 PM" width="388" src="https://user-images.githubusercontent.com/56901619/111379500-97bf6600-8679-11eb-82ea-443cb0f0ead6.png"> This is mobile version checking with iPhone 11
> 
> As you see the other buttons are not clickable.

There was a issue, we fixed it.

hiran11 commented:
@aditilamkhede  No the behaviour of  listing the post is wrong.
Sceanario 1 
 Click on Request option 
 3  post under request will be displayed.
 Click see all under request option 
 All post which has request option will be displayed.

scenario 2

Click on offer option .
 3  post under offer option will be displayed.
  click on see all under offer option 
 Then all post which has offer option to be displayed but control goes to request option 
 and all post which has request option is displayed. 
  
 Please check my attachement 

https://screencast-o-matic.com/watch/creD0qVcKVu

aditilamkhede committed:
Styling issue with NoDisplay

aditilamkhede commented:
> @aditilamkhede No the behaviour of listing the post is wrong.
> Sceanario 1
> Click on Request option
> 3 post under request will be displayed.
> Click see all under request option
> All post which has request option will be displayed.
> 
> scenario 2
> 
> Click on offer option .
> 3 post under offer option will be displayed.
> click on see all under offer option
> Then all post which has offer option to be displayed but control goes to request option
> and all post which has request option is displayed.
> 
> Please check my attachement
> 
> https://screencast-o-matic.com/watch/creD0qVcKVu

Hello, if I am wrong, you mean that when you click on see-all and select Offers tab and go back to mobile view it changes back to Request Tab?

aditilamkhede commented:
> @aditilamkhede @clubkemp @linadinh2602
> 
> 1. In the Profile page i could see "No post to display" alignment has to be fixed.
> 
> ![image](https://user-images.githubusercontent.com/51660881/111376653-1b775380-8676-11eb-847e-ef0c70824e8d.png)
> 
> 1. I could see blank space after the last post in the Request option  before reaching the end of page. Please look at the link for reference.
> 
>  Recording.13.mp4 
> ![Screenshot (126)](https://user-images.githubusercontent.com/51660881/111379489-942bdf00-8679-11eb-9a28-912309e3bf80.png)
> 
> Mobile version
> 
> Alignment of "No Posts matching your criteria " has to be fixed.
> 
> ![Screenshot_20210316-173419 (1)](https://user-images.githubusercontent.com/51660881/111384418-d9eba600-867f-11eb-8f86-cb2a4c044a4d.png)

styling issue is fixed.

aditilamkhede committed:
Merge pull request #2280 from FightPandemics/feature/fix_double_fetch

#2262 - Fix fetch calling again when go back

hiran11 commented:
> > @aditilamkhede No the behaviour of listing the post is wrong.
> > Sceanario 1
> > Click on Request option
> > 3 post under request will be displayed.
> > Click see all under request option
> > All post which has request option will be displayed.
> > scenario 2
> > Click on offer option .
> > 3 post under offer option will be displayed.
> > click on see all under offer option
> > Then all post which has offer option to be displayed but control goes to request option
> > and all post which has request option is displayed.
> > Please check my attachement
> > https://screencast-o-matic.com/watch/creD0qVcKVu
> 
> Hello, if I am wrong, you mean that when you click on see-all and select Offers tab and go back to mobile view it changes back to Request Tab?

When i click see all option under Offer tab control goes to request and it displays all request post ,Can you check the video

rabbi08 commented:
@linadinh2602 @aditilamkhede @clubkemp 

1. the 'message' icon issue is fixed and is working as expected now: PASS

2. For 'See All' in mobile:
If user scroll on the screen to navigate from 'Active -to - Archive' TAB, then no data is displayed even when the data exist. When user clicks on 'Active' or 'Archive' Tabs, to navigate to them, then data is displayed as expected.
Please see below GIF for the above issue.

![ezgif com-video-to-gif-3](https://user-images.githubusercontent.com/67168400/111399581-0a413d80-869c-11eb-9ed2-a6976b05cd08.gif)

linadinh2602 committed:
Change to use onChange

aditilamkhede committed:
Change Se-All link style

rabbi08 commented:
@aditilamkhede @linadinh2602 @clubkemp 

I have observed following issues:
- While creating new post, the timestamp is not displayed properly on user profile page though it is displayed fine on post detail page for the same post. 
- When user EDIT the post, the post is not updated on user profile page, though it is updated on post detail page.

Please see below GIF for the same.
![ezgif com-video-to-gif-4](https://user-images.githubusercontent.com/67168400/111413289-5e581c00-86b4-11eb-87f8-d4e7ca7ecd4f.gif)

- On Mobile -> iPhone 11 Pro max:
After user navigate to user profile page, nothing is clickable and the page got hang while trying. trying to reload and doing again but still not able to click anywhere on the page.

Please see below GIF

![ezgif com-video-to-gif-5](https://user-images.githubusercontent.com/67168400/111414166-e12da680-86b5-11eb-84bf-0d2e214fafaf.gif)

rabbi08 commented:
- User is able to 'LIKE' the post help board and post detail page but cannot like/click on like on 'User Profile' page.
- 

https://user-images.githubusercontent.com/67168400/111414932-48982600-86b7-11eb-9ad6-8ebb65e13c00.mov

clubkemp committed:
apply eslint ignore to postTabCard useEffect. Ignoring out future feature menu items. Restyling wrapper for just the two items

clubkemp committed:
fixing z-index of create post button

linadinh2602 committed:
temp

clubkemp committed:
updating dependency list based on latest-greatest

linadinh2602 committed:
fixing edit and likes update

clubkemp committed:
Merge pull request #2282 from FightPandemics/feature/profile-redesign-menu-edits

Feature/profile redesign menu edits

linadinh2602 committed:
backend changes

linadinh2602 committed:
update reducers

linadinh2602 committed:
ressolve updating issue

linadinh2602 committed:
add proper back from post page

aditilamkhede committed:
Fixed Active/Archived issue by setting InternalTab to name of the tab

Fixed Active/Archived issue by setting InternalTab to name of the tab. i.e. key is ACTIVE_REQ set the Inetrnaltab value to "ACTIVE"

aditilamkhede committed:
Post/Offers tab refresh issue

aditilamkhede committed:
remove unnecessary padding

aditilamkhede committed:
Merge pull request #2283 from FightPandemics/feature/fix_edit_save

#2262 - Feature/fix Edit and Likes Update

rabbi08 commented:
Thanks @aditilamkhede @clubkemp @linadinh2602 

The feature is looking good, just wanna confirm on below points on Mobile version:

- No timestamp is displayed for the post on user profile page
- No like/comment/share/message options are displayed on the post, on the user profile page, they have to navigate to post details page to do so.

Thanks

sarahjliu commented:
> Thanks @aditilamkhede @clubkemp @linadinh2602
> 
> The feature is looking good, just wanna confirm on below points:
> 
> * No timestamp is displayed for the post on user profile page
> * No like/comment/share/message options are displayed on the post, on the user profile page, they have to navigate to post details page to do so.
> 
> Thanks

@rabbi08 If you are referring to the mobile version, then it is correct that the exact time stamp, like, edit, etc will not appear on the compact posts card since it is meant to give a brief overview of the posts. Those kinds of details can be viewed on the see all page or posts detail page. In the desktop version, all these options should be available in the regular profile view because there is more space and no need for a compact posts card.

rabbi08 reviewed (dismissed):
Thanks @sarahjliu

aditilamkhede committed:
Merge branch 'staging' into feature/profile-redesign
